langchain4j-milvus/src/main/java/dev/langchain4j/store/embedding/milvus/MilvusEmbeddingStore.java

```diff
@@ -116,21 +123,54 @@ public String add(Embedding embedding, TextSegment textSegment) {
 
   public List<String> addAll(List<Embedding> embeddings) {
     List<String> ids = generateRandomIds(embeddings.size());
-    addAllInternal(ids, embeddings, null);
+    addAllInternal(ids, embeddings, null, null);
     return ids;
   }
 
   public List<String> addAll(List<Embedding> embeddings, List<TextSegment> embedded) {
     List<String> ids = generateRandomIds(embeddings.size());
-    addAllInternal(ids, embeddings, embedded);
+    addAllInternal(ids, embeddings, embedded, null);
+    return ids;
+  }
+
+public String add(Embedding embedding, String partitionName) {
```

Comment:
I am not sure duplicating each method in order to specify partition is the right way to go.
What about setting partition in the constructor/builder?

Reply:
After obtaining a connection, different partitions may be operated during multiple operations. At the same time, it is also convenient to expand into a connection pool in the future to maintain connections.

Reply:
@igerdy this approach is definitely not scalable. I propose to propagate partition via `EmbeddingStoreRequest` instead. We could add a `Map<String, Object>` field that will hold store-specific properties, then `MilvusEmbeddingStore` could read partition from it.

WDYT?

Reply:
Your idea is to add a class in the `core` module to cover the features of different `EmbeddingStore`?
Good suggestion, this will make the extensibility stronger.
But I think the possibility of multiple `EmbeddingStore` in a project is relatively low. Can it be extended by extending the interface and implementation in the specific EmbeddingStore module? What do you think?